man/man1/mash-job-add.1

```diff
@@ -1,8 +1,92 @@
+.\" Process this file with
+.\" groff -man -Tascii mash-job-add.1
+.\"
 .TH "MASH JOB ADD" "1" "27-Aug-2018" "" "mash job add Manual"
 .SH NAME
-mash\-job\-add \- Send add job request to mash server based on...
+mash\-job\-add \- Add a job to the mash server pipeline
 .SH SYNOPSIS
 .B mash job add
-[OPTIONS] DOCUMENT
+[OPTIONS]
 .SH DESCRIPTION
-Send add job request to mash server based on provided json document.
\ No newline at end of file
+Create a job on the \fImash\fP server for execution in the pipeline.
+.SH OPTIONS
+.TP
+\fB--account\fP
+The name of the account to be used for image testing or release. See
+\fBmash-account-add(1)\fP for details. The value may also specify a
+group name.
+.TP
+\fB--allow-copy\fP ALLOW_COPY
+Flag to enable the account(s) with which the image is shared to copy the image
+by allowing copy access to the underly The option only applies
+when the \fIFRAMEWORK\fP is set to \fiec2\fP.
+.TP
+\fB--distro\fP DISTRO
+A short name for the distribution. Supported values
+\fIopensuse\P and \fIsles\fP
+.TP
+\fB--framework\fP FRAMEWORK
+Specify the target cloud framework for this job. Supported values are
+\fIazure\fP, \fIec2\fP, \fIgce\fP.
+.TP
+\fB--image\fP KIWI_IMAGE_NAME
+The name for the image as set in the KIWI image config file,
+\fIconfig.xml\fP or \fIconfig.kiwi\fP
+.TP
+\fB--image-description\fP IMAGE_DESCRIPTION
+A description for the image, this will be added in the cloud framework.
+.TP
+\fB--image-name\fP IMAGE_NAME
+The name of the image as it is to be named in the cloud framework.
+.TP
+\fB--image-url\fP IMAGE_URL
+The full url from where the image can be downloaded.
+.TP
+\fB--job-doc\fP JOB_DOCUMENT
+Path to a completed job document. The job document describes the job the
+\fImash\fP server is supposed to add to it's pipeline for execution.
+.TP
+\fB--no-delete\fP NO_DELETE
+When a job is submitted for testing purposes the default is to delete
+the image form the framework if the tests pass. This option preserves
+the image in the framework.
+.TP
+\fB--old-image-name\fP OLD_IMAGE_NAME
+The name of the image in the cloud framework being replaced by this new
+image.
+.TP
+\fB--partition\fP PARTITION_NAME
```

Comment:
Should not be option here. There's no way to override an accounts partition. This is already set when an account is created.